smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/swiftmodules/SwiftSymbol.kt

```diff
@@ -12,11 +12,13 @@ class SwiftSymbol {
             name: String,
             declaration: SwiftDeclaration?,
             dependency: Dependency?,
-            spiName: String?
+            additionalImports: List<Symbol>,
+            spiNames: List<String>,
```

Comment:
The factory method in SwiftSymbol adds parameters for additional imports, and changes the spiName import from an optional String to a list of Strings.